iothub/device/src/Pipeline/DefaultDelegatingHandler.cs

```diff
@@ -154,7 +154,7 @@ protected void ThrowIfDisposed()
             }
         }
 
-        protected virtual void Dispose(bool disposing)
+        protected private virtual void Dispose(bool disposing)
```

Comment:
We can use `protected private` in more places, for internal classes that have protected members.